lib/rules/no-exclusive-tests.js

```diff
@@ -23,12 +23,22 @@ module.exports = function (context) {
            isPropertyNamedOnly(callee.property);
     }
 
+    function createAutofixFunction(callee) {
+        return function removeOnlyProperty(fixer) {
+            return fixer.removeRange([ callee.object.range[1], callee.range[1] ]);
```

Comment:
This could be improved by extracting variables for the range to make clear, what `callee.object.range[1]` & `callee.range[1]` exactly are but something as general as an `only` function is really hard to name I guess. So unfortunately nothing came to my mind. Do you have an idea?